xwiki-platform-core/xwiki-platform-velocity/xwiki-platform-velocity-webapp/src/main/java/org/xwiki/velocity/XWikiWebappResourceLoaderInitializer.java

```diff
@@ -48,8 +48,5 @@ public void initialize(Properties velocityProperties)
         velocityProperties.setProperty(
             RuntimeConstants.RESOURCE_LOADER + '.' + RESOURCE_LOADER_ID + '.' + RuntimeConstants.RESOURCE_LOADER_CLASS,
             XWikiWebappResourceLoader.class.getName());
-
-        // Add macros.vm as default template
-        velocityProperties.put(RuntimeConstants.VM_LIBRARY, "/templates/macros.vm");
```

Comment:
Why is it not needed anymore?

Reply:
Ah found it, you put it in https://github.com/xwiki/xwiki-platform/pull/2331/files#diff-c35980baa623937175da7e26c81623cdd604dabae2f84d1810f58fae00b17be6R311

Reply:
Yes, I have to completely think the support of macros.vm, as macros provided at Runtime init have priority over those passed as libraries (does not make much sense to be...). The good thing is that it will be much easier to support modifying macros.vm at runtime (right now, if you modify macros.vm and refresh, it's still using the old one).